### ISSUE TITLE
CatalogTitles: show more then 10 results, if available

### ISSUE THREAD

**Issue by nichtich** [gbv/coli-ana#58]
Query with limit=11 and allow to increase the limit by clicking on "..." as last list item.

**Comment by stefandesu** [comment]
I can try to implement this, but there are some issues:
- Note that it uses the parameter `count` instead of `limit` (see http://ws.gbv.de/suggest/csl2?info).
- There is no equivalent to an `offset` parameter, so the only thing we can do is increase the `count` parameter and load again.
- The order of results seems to be scrambled and there is no sorting parameter. This means that if you compare the requests with `count=11` and with `count=21`, they will not share the first 11 results. Instead, further results are inserted seemingly at random into the result set.

We can still implement it the same way, but 1) clicking on "..." will not append new results, and 2) the more results the user loads, the bigger the data set will get (I don't think this will become much of an issue though).

**Comment by nichtich** [comment]
Just reload the whole result set when a larger number is requested.

**Comment by stefandesu** [comment]
Now implemented. Should we add some kind of loading indicator? Right now the results load so fast that it's not necessary, but it would be bad interface design if it loaded for a few seconds without any indicator after clicking "load more".

**Comment by nichtich** [comment]
Yes, loading indicator would be good, we already have the Vue component :-)
